examples/coverage/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20-buster
+FROM golang:1.21-buster
```

Comment:
IIRC they don't make `buster` anymore, so this one might have to upgrade to bullseye?